### DISCUSSION TITLE
Macos big sur: Problem with manager.py view

### DISCUSSION THREAD

**Discussion by azhe** [swar/Swar-Chia-Plot-Manager#616] [Q&A]
Progress is not displayed but plotting is running fine.
I have tried "sudo python manager.py view" but the problem still not solved.

![20210520154936](https://user-images.githubusercontent.com/1940482/118941635-81d56680-b984-11eb-9ef7-3995a3b83bbf.png)
![20210520155150](https://user-images.githubusercontent.com/1940482/118941668-8b5ece80-b984-11eb-9c3a-edea34b495e5.png)

**Comment by ludgats** [comment]
What Mac are you running on?

**Reply by azhe** [reply]
Actually, Im using HP400G3 Desktop mini with Hackintosh and the CPU is QL3X(7820HK es).
Maybe this program does not support Hackintosh very well? - -
